pkg/controller/deployments/controller.go

```diff
@@ -49,6 +50,7 @@ func newReconciler(mgr manager.Manager, k8sClient kubernetes.Interface,  riServi
 			deploymentClient: k8sClient.AppsV1(),
 		},
 		podService:       cluster.NewPods(mgr.GetClient()),
+		imageService:clusterImageService,
```

Comment:
Formatting looks off here. Maybe we should just add the code fix/check make targets to we can fix these https://github.com/integr8ly/integreatly-operator/blob/master/Makefile#L76-L82